.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,204 @@
+Contributing
+-------------
+
+Thank you for your interest in contributing to PHP_CodeSniffer!
+
+
+## Reporting Bugs
+
+Please search the [open issues](https://github.com/squizlabs/PHP_CodeSniffer/issues) to see if your issue has been reported already and if so, comment in that issue if you have additional information, instead of opening a new one.
+
+Before reporting a bug, you should check what sniff an error is coming from.
+Running `phpcs` with the `-s` flag will show the name of the sniff for each error.
+
+If the error code starts with anything other than `Generic`, `MySource`, `PEAR`, `PSR1`, `PSR2`, `PSR12`, `Squiz` or `Zend`, the error is likely coming from an external PHP_CodeSniffer standard.
+**Please report bugs for externally maintained sniffs to the appropriate repository.**
+
+Bug reports containing a minimal code sample which can be used to reproduce the issue are highly appreciated as those are most easily actionable.
+
+:point_right: Reports which only include a _screenshot_ of the code will be closed without hesitation as not actionable.
+
+
+### Reporting Security Issues
+
+PHP_CodeSniffer is a developer tool and should generally not be used in a production environment.
+
+Having said that, responsible disclosure of security issues is highly appreciated.
+Issues can be reported privately to the maintainers by opening a [Security vulnerability report](https://github.com/squizlabs/PHP_CodeSniffer/security/advisories/new).
+
+
+### Support/Questions About Using PHP_CodeSniffer
+
+Please read the [documentation in the wiki](https://github.com/squizlabs/PHP_CodeSniffer/wiki) before opening an issue with a support question.
+
+The [discussion forum](https://github.com/squizlabs/PHP_CodeSniffer/discussions) or [StackOverflow](https://stackoverflow.com/questions/tagged/phpcodesniffer) are the appropriate places for support questions.
+
+
+## Contributing Without Writing Code
+
+### Issue Triage
+
+We welcome issue triage.
+
+Issue triage is the action of verifying a reported issue is reproducible and is actually an issue.
+It includes checking whether the issue is something which should be fixed in **_this_** repository.
+
+To find issues which need triage, look for [issues without labels](https://github.com/squizlabs/PHP_CodeSniffer/issues?q=is%3Aissue+is%3Aclosed+no%3Alabel) or issues with the ["Status: triage"](https://github.com/squizlabs/PHP_CodeSniffer/labels/Status%3A%20triage) label.
```

Comment:
I think this link is wrong as it's showing closed issues. Did you mean to use https://github.com/squizlabs/PHP_CodeSniffer/issues?q=is%3Aissue+no%3Alabel+is%3Aopen

Reply:
Thanks for catching that! Fixed now.